README.md

```diff
@@ -1,5 +1,71 @@
-chef-acceptance
----------------
+`chef-acceptance` helps develop and run acceptance tests from your laptop and from Chef Delivery.
 
-`chef-acceptance` is an experimental framework for running acceptance tests
-within a Chef product's source project.
+## Why
+
+`chef-acceptance` makes it easy to develop acceptance tests for a project. You can run any type of suites for your project (test-kitchen, pedant, chef provisioning, rspec, etc...) but it enforces a structure so that you can run all of your different acceptance tests in the same way. It also gives you a CLI interface so that you can build your tests without breaking your pipeline until they are ready.
+
+## Usage
+
+Add `./acceptance` directory to your project with the following directory structure:
+
+```
+acceptance
+├── lamont-ci
+│   ├── .acceptance
+│   │   ├── metadata.rb
+│   │   └── recipes
+│   │       ├── destroy.rb
+│   │       ├── provision.rb
+│   │       └── test.rb
+│   ├── .kitchen.yml
+│   ├── metadata.rb
+│   └── recipes
+│       └── default.rb
+└── spincycle
+    ├── .acceptance
+    │   ├── metadata.rb
+    │   └── recipes
+    │       ├── destroy.rb
+    │       ├── provision.rb
+    │       └── test.rb
+    ├── .kitchen.yml
+    ├── metadata.rb
+    └── recipes
+        └── default.rb
+```
+
+Now you can run your acceptance tests as below:
+
+```
+# chef-acceptance <subcommand> <suite-name> [options]
+chef-acceptance provision spincycle
+chef-acceptance test spincycle
+chef-acceptance destroy spincycle
+```
```

Comment:
I think we can capture this and implement it later on since it will bring in some concerns for concurrency.